h/static/scripts/account/test/auth-controller-test.coffee

```diff
@@ -18,15 +18,36 @@ class MockSession
     $promise:
       finally: sandbox.stub()
 
-mockFlash = info: sandbox.spy()
-mockFormRespond = sandbox.spy()
-
-describe 'h:AuthController', ->
+# Return an AuthController instance and associated objects for tests to use.
+_controller = (mocksession) ->
+  controller = null
   $scope = null
   $timeout = null
-  auth = null
   session = null
-  $controller = null
+  mockFlash = info: sandbox.spy()
+  mockFormRespond = sandbox.spy()
+  inject(($controller, $rootScope) ->
+    session = mocksession or new MockSession()
+    $scope = $rootScope.$new()
+    $timeout = sandbox.spy()
+    params =
+      $scope: $scope
+      $timeout: $timeout
+      flash: mockFlash
+      session: session
+      formRespond: mockFormRespond
+    controller = $controller('AuthController', params)
+  )
+  return(
```

Comment:
```coffeescript
return(
  foo: bar
)
```

seems a bit perverse to me

```coffeescript
return {
  foo: bar
}
```

is, well, JavaScript, more-or-less, and would be preferred.